index.js

```diff
@@ -15,21 +15,23 @@ function unlinkOrRmrfSync(path) {
     rimraf.sync(path);
   }
 }
+var ASSIMILATION_SYMBOL = 'broccoli-merge-tree@1.0.0';
```

Comment:
doesn't need to be semver, a seperate ASSIMILATION protocol versions is likely best. Allowing for multiple version collapses..